models/post.coffee

```diff
@@ -0,0 +1,14 @@
+mongoose = require('mongoose')
```

Comment:
What do you mean not inlining it? It is in *server.coffee* because it needs to connect to the database.